src/h2olog/h2olog.cc

```diff
@@ -39,11 +41,18 @@ static void usage(void)
     printf(R"(h2olog (h2o v%s)
 Usage: h2olog -p PID
        h2olog quic -p PID
-       h2olog quic -s response_header_name -p PID
-Other options:
-    -h Print this help and exit
+Optional arguments:
     -d Print debugging information (-dd shows more)
+    -h Print this help and exit
+    -l Print the list of handled USDTs and exit
+    -s RESPONSE_HEADER_NAMES Response header names to show, e.g. "content-type"
```

Comment:
Should use singular, as we accept only one value as an argument. We can separately state that the option can be repeated multiple times.